akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -12,6 +12,9 @@ akka.serialization.jackson {
   jackson-modules += "akka.serialization.jackson.AkkaJacksonModule"
   # AkkaTypedJacksonModule optionally included if akka-actor-typed is in classpath
   jackson-modules += "akka.serialization.jackson.AkkaTypedJacksonModule"
+  // FIXME how does that optinal loading work??
```

Comment:
Here: https://github.com/akka/akka/blob/master/akka-serialization-jackson/src/main/scala/akka/serialization/jackson/JacksonObjectMapperProvider.scala#L137

Reply:
Ah, thanks, I couldn't figure it out yesterday.